readme.md

```diff
@@ -34,6 +34,10 @@ Unlike PSPDFKit server, it catches errors when document with provided ID was alr
 See [Client Authentication](https://pspdfkit.com/guides/server/current/pspdfkit-server/client-authentication/)
 section of PSPDFKit documentation for details.
 
+## Publish
+
+Run `sudo npx np --release-draft-only`, follow interactive UI. Please select appropriate version bump according to [semver](https://semver.org/).
```

Comment:
It'll publish and redirect to newly created release in the repo
```suggestion
Run `sudo npx np`, follow interactive UI. Please select appropriate version bump according to [semver](https://semver.org/).
```

Reply:
Also, let's add second step, like: "- After redirect to prepared release just press button `Publish release`"